server/app/assets/javascripts/sort_selector.ts

```diff
@@ -1,7 +1,7 @@
 /**
- * Responsible for controlling the sorting mechanism used by QuestionBankController. Elements (tagged with elementSelector class)
- * have data attributes with the value to be compared and will sort list using the value. The data attribute is defined
- * in QuestionSortOption.
+ * Responsible for controlling the sorting mechanism used by QuestionBankController and TrustedIntermediaryController.
+ * Elements (tagged with elementSelector class) have data attributes with the value to be compared and will sort
+ * list using the value. The data attribute is defined in QuestionSortOption.
  *
  * E.g. Sort by Name A-Z will look up the value for data attribute tiname-asc and sort the list by that value.
```

Comment:
Is it weird that `tiname-asc` was already used as the example in this comment? This functionality is new, right? I'm wondering if I'm missing something.

Reply:
I extracted the method which was previously in question bank controller to this new shared method